AwesomeTextField/Classes/AwesomeTextField.swift

```diff
@@ -10,104 +10,104 @@ import UIKit
 
 @IBDesignable
 open class AwesomeTextField: UITextField {
-
+    
```

Comment:
Please, remove trailing whitespaces on blank lines.
The rest looks good to merge